strings/strings.c

```diff
@@ -20,7 +24,13 @@ int string_length(char s[])
 */
 char *reverse_string(char rv[], char s[])
 {
-
+    int slen = string_length(s);
+    rv[slen] = '\0';
+    for (int i = 0; s[i] != '\0'; i++) {
+        slen -= 1;
+        rv[slen] = s[i];
```

Comment:
I like this creative implementation. Seems like you have a pretty good grasp on C strings.